src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -216,4 +219,19 @@ private OssFeatures() {
    */
   public static final Feature<Boolean> SIGNS_ARTIFACTS
       = new BooleanFeature("If a project signs artifacts");
+
+  /**
+   * If OWASP Dependency plugin is used to scan the project. It is either used as a mandatory step,
+   * optional step or not used at all.
+   */
+  public static final Feature<Status> OWASP_DEPENDENCY_CHECK_SCAN_AVAILABILITY =
+      new AvailabilityFeature("OWASP Dependency Check scan mode");
```

Comment:
At the moment, I am not sure if we're going to use `Status` and `AvailabilityFeature` in other places. Therefore let's please make them specific. Please rename `Status` to `OwaspDependencyCheckUsage`. Then, let's please rename `AvailabilityFeature` to `OwaspDependencyCheckUsageFeature`. I know it is quite a long name and looks a bit ugly but I believe it has to be specific here. Finally, please rename `AvailabilityValue` to `OwaspDependencyCheckUsageValue` accordingly.

In fact, we have the `EnumValue` class. But unfortunately Jackson fails to deserialize it. This disabled test shows reproduces the issue

https://github.com/SAP/fosstars-rating-core/blob/master/src/test/java/com/sap/sgs/phosphor/fosstars/model/value/EnumValueTest.java#L88

I am not sure if newer versions of Jackson work fine but https://github.com/FasterXML/jackson-databind/issues/2605 seems to be fixed. So, you can try to use the `EnumValue` instead of introducing `OwaspDependencyCheckUsageValue`.

Reply:
Not sure why but `EnumValue` still fails. Updated `jackson-databind to 2.11.1`

`com.fasterxml.jackson.databind.exc.InvalidDefinitionException:
Cannot construct instance of `java.lang.Enum` (no Creators, like default construct, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
 at [Source: (BufferedInputStream); line: 38, column: 12] (through reference chain: com.sap.sgs.phosphor.fosstars.model.qa.TestVectors["elements"]->java.util.ArrayList[1]->com.sap.sgs.phosphor.fosstars.model.qa.StandardTestVector["values"]->java.util.HashSet[0]->com.sap.sgs.phosphor.fosstars.model.value.EnumValue["value"])`

Reply:
Maybe the issue still exists, maybe some annotation need to be added. Anyway. we can work on in it later.

Reply:
https://github.com/SAP/fosstars-rating-core/issues/273